File: primenumber.c
include <stdio.h>
//Prime number between 1 to n:
int main(void)
{
int i,j,count,a;
printf("Enter the number:");
scanf("%d",&a);
printf("\n\tPrime numbers between 1 to %d",a);
printf("\n");
  for(i=1;i<=a;i++)
    {
      count=0;
      for(j=1;j<=i;j++)
        {
          if(i%j==0)
          {
            count++;
          }
        }
    if(count==2)
    {
      printf("\n%d",i);
    }
    }
  return 0;
}

File: primenumber.py
#Prime number between 1 to n
num=int(input("Enter the number:"))
for i in range(1,num+1):
  count=0
  for j in range(1,i+1):
    if i%j==0:
      count=count+1
  if count==2:
    print(i)
